src/openrct2/actions/RideEntranceExitPlaceAction.hpp

```diff
@@ -129,7 +127,7 @@ DEFINE_GAME_ACTION(RideEntranceExitPlaceAction, GAME_COMMAND_PLACE_RIDE_ENTRANCE
         auto res = MakeResult();
         res->Position.x = _loc.x + 16;
         res->Position.y = _loc.y + 16;
-        res->Position.z = tile_element_height(_loc);
+        res->Position.z = z;
```

Comment:
Is this the same as `ride->stations[_stationNum].Height * 8` ? They look like very different functions, I'm confused.

Reply:
Not sure why it was doing tile element height as it is the wrong height. It should have been station height from the beginning

Reply:
How curious, I wonder if it fixes any bugs. Thanks for explaining :)